tasks/word games.md

```diff
@@ -0,0 +1,57 @@
+# Summary
+Participants were given one word and asked to come up with as many new words as they could using the letters from that one word in ten minutes. 
+
+# References
+Main Paper: http://collections.uakron.edu/digital/collection/p15960coll1/id/21257/
+
+# Stimuli
+## The visual components
+A card having different words printed on it. Words included: 
+- neurotics 
+- courtesan 
+
+## Materials for alternative versions of the experiment 
+N/A
+
+# Procedure
+## Steps
+1. 49 judges who sorted 104 tasks into eight categories (or piles) representing different positions on the dimension under consideration. 
+2. Each judge sorted the tasks on all dimensions, but no one sorted tasks on more than one dimension on a given day, and usually the sortings were separated by several days. 
+3. The orders in which dimensions were sorted by the various judges were random, and the set of tasks was shuffled by E before each sorting. 
+4. Each task was typed on a separate sheet which included all the information  
+5. For a given dimension, the judge was given the set of tasks, a definition of dimension (exactly as given above), and a copy of the Archives of the History of American Psychology, The Center for the History of Psychology, The University of Akron CHP Special Interest Collection 
+
```

Comment:
Maybe we can add:
6. When the sorting had been completed, the judge placed each pile of tasks in an appropriately labeled manila envelope and returned them to E, who recorded the responses.